samples/military_feature_schemas/Fields_AirMissile.csv

```diff
@@ -0,0 +1,32 @@
+field_name,field_type,field_length,field_alias,nullability,field_domain,field_default,field_setsubtypes, field_subtype, field_notes
+identity,Integer,4,Standard Identity,NULLABLE,Identity,,FALSE,,
+context,SmallInteger,2,Context,NULLABLE,Context,,FALSE,,
+symbolset,SmallInteger,2,Symbol Set,NULLABLE,Air_Missile,,FALSE,,
```

Comment:
Air_Missile or AirMissile - it looks like this might not match the convention of the other symbol set domains (not that there is a consistent convention, but it looks like only the entities and modfiers had "_"'s)